test/e2e/setup/test_config.go

```diff
@@ -36,8 +36,8 @@ func LoadTestConfigFromEnv() TestConfig {
 		CertManagerNamespace:    envvar.GetEnvOrDefault(testCertManagerNamespaceEnvName, "cert-manager"),
 		CertManagerVersion:      envvar.GetEnvOrDefault(testCertManagerVersionEnvName, "v1.5.3"),
 		OperatorImage:           envvar.GetEnvOrDefault(operatorImageEnvName, "quay.io/mongodb/community-operator-dev:latest"),
-		MongoDBImage:            envvar.GetEnvOrDefault(construct.MongodbName, "mongodb-community-server"),
-		MongoDBRepoUrl:          envvar.GetEnvOrDefault(construct.MongodbRepoUrl, "docker.io/mongodb"),
+		MongoDBImage:            envvar.GetEnvOrDefault(construct.MongodbImageEnv, "mongo"),
```

Comment:
I believe we want to use the official image here shipped by Mongo.

Reply:
Sure! I was thinking of making it more consistent since the helm charts use these images instead. But for testing it doesn't make a difference.